src/ui/public/vis/editors/default/vis_editor_resizer.js

```diff
@@ -0,0 +1,58 @@
+import $ from 'jquery';
+import { uiModules } from 'ui/modules';
+import { keyCodes } from 'ui_framework/services';
+
+uiModules
+  .get('kibana')
+  .directive('visEditorResizer', function () {
+    return {
+      restrict: 'E',
+      link: function ($scope, $el) {
+        let $left;
+        const minWidth = 360;
+
+        $el.on('mousedown', function (event) {
+          if (!$left) {
+            $left = $('.collapsible-sidebar');
```

Comment:
Maybe we should use the `<vis-editor-sidebar>` element instead of the encapsulating `.collapsible-sidebar` div to rescale. The `vis-editor-sidebar` already has a `min-width` and the `width` said. If we use this (and of cause also apply the default editor size classes to it, we could also get rid of the `minWidth` checks in this class and thus not need to have hardcoded width values in JavaScript, since the `min-width` in CSS of the `<vis-editor-sidebar>` would work fine otherwise.

Another solution would be, pulling up the styling from `<vise-editor-sidebar>` to the `.collapsible-sidebar` div, but I think the actual editor sidebar would be the more reasonable element.

Reply:
In whatever way we decide, we maybe want to limit this element finding to a bit more local scope, e.g. `$el.siblings('vis-editor-sidebar')`

Reply:
we can't then set the size correctly as the parent of vis-editor-sidebar is the collapsible-sidebar ... 
moving min-width to collapsible-sidebar instead as thomas suggested